### PR TITLE
Universal Robots 

### PR DESCRIPTION
**Pull request by DavidMansolino**
Improvements from the URDF:
  - [x] Improved Meshes.
  - [x] Improved bounding-objects.
  - [x] Fixed masses enad center of masses.
  - [x] Doc:
    - https://www.cyberbotics.com/doc/guide/actuators?version=universal-robot
    - https://www.cyberbotics.com/doc/guide/gripper-actuators?version=universal-robot#robotiq-3f-gripper
    - https://www.cyberbotics.com/doc/guide/robots?version=universal-robot
    - https://www.cyberbotics.com/doc/guide/ur5e?version=universal-robot
  - [x] Sample.
  

empty_2
(with a ROBOTIQ arm to test)

### CODE REVIEW

**Comment by DavidMansolino** [comment]
Current status of the sample (still need some improvements):
ezgif-4-84a445c72118

**Comment by DavidMansolino** [comment]
New version with performance issue fixed and scene improvements:
ur

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:+1:

**Comment by fabienrohrer** [comment]
👀

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

File: docs/guide/object-factory.md [hunk 1]
```diff
@@ -169,6 +172,41 @@ FireExtinguisher {
...
+> **File location**: "WEBOTS\_HOME/projects/objects/factory/manhole/protos/SquareManhole.proto"
+
+> **License**: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+[More information.](https://cyberbotics.com/webots_assets_license)
+
+#### SquareManhole Field Summary
+
+- `size`: Defines the size of the manhole.
+
+- `enableBoundingObject`: Defines whether the fire extinguisher should have physics.
```
Comment (omichel) [inline docs/guide/object-factory.md]: fire extinguisher? physics?

File: docs/guide/ur5e.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+## Universal Robot UR5e
+
+%robot ur5e images/robots/ur5e/model.png
+
+The [Universal Robot UR5e](https://www.universal-robots.com/products/ur5-robot/) is a flexible collaborative robot arm with 6 degree of freedom.
```
Comment (omichel) [inline docs/guide/ur5e.md]:
```suggestion
The [Universal Robot UR5e](https://www.universal-robots.com/products/ur5-robot/) is a flexible collaborative robot arm with 6 degrees of freedom.
```

File: docs/guide/ur5e.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+- `staticBase`: Defines if the robot base should be pinned to the static environment.
+
+### Samples
+
+You will find the following sample in this folder: "WEBOTS\_HOME/projects/robots/universal_robots/ur5e/worlds".
+
+#### ur5e.wbt
+
+![ur5e.wbt.png](images/robots/ur5e/ur5e.wbt.png) This simulation shows two UR5e robots equipped with a [ROBOTIQ 3F Gripper](gripper-actuators.md#robotiq-3f-gripper) grabbing can on a conveyor belt and putting them in crates.
```
Comment (omichel) [inline docs/guide/ur5e.md]:
```suggestion
![ur5e.wbt.png](images/robots/ur5e/ur5e.wbt.png) This simulation shows two UR5e robots equipped with a [ROBOTIQ 3F Gripper](gripper-actuators.md#robotiq-3f-gripper) grabbing cans on a conveyor belt and putting them in crates.
```

File: docs/guide/object-factory.md (hunk 1 above)
Comment (DavidMansolino) [reply docs/guide/object-factory.md]: Oups... I copy pasted the PROTO and forget to update this field description.

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
I tested to use a normal map instead of geometry for the logo and it works fine as long as we don't zoom too much:
(normal map: left | geometry: right):
ur5e_1
ur5e_2
When zooming in because of the texture resolution it is not so nice compared to the geometry one (but since you have to zoom a lot I think it fine):
ur5e_3 (normal map)
ur5e_4 (geometry)

**Comment by omichel** [comment]
It looks better to me with this normal map. Adding the ambient occlusion map might be even better.

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

**Pull request merged**